src/ui/engine/header.ts

```diff
@@ -96,14 +96,14 @@ export function updateHeader(node: HTMLElement, col: Column, ctx: IRankingHeader
   if(sort) {
     const {asc, priority} = col.isSortedByMe();
     sort.dataset.sort = asc !== undefined ? asc : '';
-    sort.dataset.priority = priority !== undefined ? priority : '';
+    sort.dataset.priority = priority !== undefined ? `${(parseInt(priority, 10) + 1)}.` : '';
```

Comment:
is it needed to add the '.' here? as far as I know it is possible to concat strings in the content attribute ala `content: attr(data-priority) '.'`

Reply:
I think using `delete sort.dataset.priority` it is possible to remove the attribute if there is no value to set, Then no `:not` selector is needed

Reply:
I addressed both with the upcoming commit. Please check again.